ckan/public-midnight-blue/base/scss/_toolbar.scss

```diff
@@ -34,7 +37,7 @@
 }
 
 .toolbar .breadcrumb .active {
-    color: $breadcrumb-active-color;
```

Comment:
This is defined in `_variables.scss` we can change it's value to $primary and use it as the color in `.toolbar .breadcrumb .active a, .toolbar .breadcrumb a.active` and remove `.toolbar .breadcrumb .active` because it doesn't do anything currently.